t/unit/transport/test_redis.py

```diff
@@ -828,6 +828,17 @@ def test_rediss_connection(self):
                     redis.redis.SSLConnection,
                 )
 
+    def test_global_key_prefix(self):
+        conn = Connection(transport=Transport, transport_options={
+            'global_key_prefix': 'foo',
+        })
+        chan = conn.channel()
+        c = chan._create_client = Mock()
+
+        body = {'hello': 'world'}
+        chan._put_fanout('exchange', body, '')
```

Comment:
I think you should also check [Channel._put](https://github.com/celery/kombu/pull/912/files#diff-4b690d9e140dd19f7d015f4595540bcaR570), not only the fanout case.

Reply:
@wetneb did you perhaps consider the above?

Reply:
@georgepsarakis I did add a test case for that, yes!